Makefile

```diff
@@ -2,7 +2,7 @@
 
 GIT_COMMIT=$(shell git rev-parse HEAD 2> /dev/null || echo unknown)
 CLC_VERSION=$(shell git describe --tags `git rev-list --tags --max-count=1` || echo UNKNOWN)
-LDFLAGS="-X 'github.com/hazelcast/hazelcast-go-client/internal.ClientType=CLC' -X 'github.com/hazelcast/hazelcast-commandline-client/internal.GitCommit=$(GIT_COMMIT)' -X 'github.com/hazelcast/hazelcast-commandline-client/internal.ClientVersion=$(CLC_VERSION)' -X 'github.com/hazelcast/hazelcast-go-client/internal.ClientVersion=$(CLC_VERSION)'"
+LDFLAGS="-X 'github.com/hazelcast/hazelcast-go-client/internal.ClientType=CLC' -X 'github.com/hazelcast/hazelcast-commandline-client/internal.GitCommit=$(GIT_COMMIT)' -X 'github.com/hazelcast/hazelcast-commandline-client/internal.Version=$(CLC_VERSION)' -X 'github.com/hazelcast/hazelcast-go-client/internal.ClientVersion=$(CLC_VERSION)'"
```

Comment:
Is it valid to replace `ClientVersion` with `Version`? We have `ClientVersion` variable in `internal/release.go` file and I thought this line set up this variable. https://github.com/yuce/hazelcast-commandline-client/blob/refactor-stage-1/internal/release.go#L6

Also, we are using this variable in `version` command so we need to refactor it as well if we select `Version`.

Reply:
Since the PRs are coming one by one for the refactor, there may be initial inconsistencies. This is what we are aiming to go: https://github.com/yuce/hazelcast-commandline-client/blob/refactor/internal/version.go